package.json

```diff
@@ -8,6 +8,7 @@
     "test": "ng test",
     "test-ci": "ng test --no-watch --no-progress --browsers=ChromeHeadlessCI --codeCoverage=true",
     "lint": "ng lint",
+    "docs": "typedoc",
```

Comment:
This is great - let's also execute this whenever we build, I suggest putting it into a `postbuild` script:

```
"postbuild": "npm run docs"
```

Reply:
Also, let's make sure we publish a link to the docs in the readme, see [Auth0 React](https://github.com/auth0/auth0-react/#documentation).

Reply:
One other thing - when you create the "postbuild" script, can you also refactor the `npm run useragent` command from the `build` script into a new `prebuild` script?

So:

```json
"build": "npm run update-useragent && ng build",
```

would become:

```json
"prebuid": "npm run update-useragent",
"build": "ng build"
```

Reply:
Done 🏁

Reply:
Awesome! And thanks for referencing the quickstart too, nice job 👏